Organisation refurbishing (#588)
datasektionen/bawang-content: sektionen/organisation/body_en.md

The Computer Science Chapter are also proud proprietors of a permanent
license to serve alcohol and thus we follow the Swedish Alcohol Law (Alkohollag:2010:1622). To be clear to all our members about our view on alcohol in our chapter hall and our activities we have the [PM for Alcohol](https://styrdokument.datasektionen.se/pm/pm_alkohol).

We also have a [Cohabitatation Agreement](https://dsekt.se/samboendeavtal) regulating how the chapter hall META is shared between The Computer Science Chapter and the Chapter for Media Technology. 

## Members

Members are everyone who studies one of the programs that belong to the chapter and who pays the [THS membership fee](https://thskth.se/en/membership). The board of directors are members, the officials are members, and we hope that you are a member! Every member is allowed to attend and vote at the chapter meetings (SM), stand for election and come to our pubs.

> ## Our meetings

Reviewer: VM seems to be missing

The chapter has two types of formal meetings: chapter meetings, where all members are able to vote and have a say, as well as board meetings, where all members are welcome to participate and express their opinion but only the board can vote. All meeting protocols can be found under [protocols](/sektionen/dokument?lang=en#protocols).

### Chapter Meeting (Sektionsmöte, SM)

Chapter meeting, or SM, is the chapter's highest decision-making body. They could be compared with the Computer Science Chapter's annual meeting, but there are multiple every year. Every member of the Computer Science Chapter has a right to speak their mind and vote at the SM. The most important decisions affecting the chapter are made during SMs. There are four
ordinary SMs in the year; Budget-SM (Budget Meeting), Glögg-SM (Christmas Meeting), Revisions-SM (Audit meeting), and

> Val-SM (Election Meeting). More chapter meetings can be arranged according to demand. If you want to suggest an alteration in the documents you leave a motion to SM. To change the by-laws it is required to be passed at two separate SMs.

Reviewer: Glass-SM?

Reviewer 2: :ice_cream:
men det har väl fortfarande inte bytt namn

Reviewer: I suppose not :p

Author: There is no such thing as Glass-SM

### D-directorate Meeting (D-rektoratsmöten, DM)

The Board of Directors Meeting. All of the chapter members have the right to attend and speak, but only the board have the right to vote. Every volunteer gets an invitation to DM but it's only the members of the D-directorate that are expected to attend. If a volunteer has sent a proposal to the board that will be handled in the meeting, they are also expected to attend.

## Activities

<img src="https://dsekt-assets.s3.amazonaws.com/organisation/verksamhetskarta.png" style="width:100%;" />

The chapter has a wide range of activities that our members can take part of. It is led by the board and carried out by our elected officials and clubs. Every official is responsible 

### D-directorate

> The D-directorate is the board of the Computer Science chapter and consists of seven members.

Reviewer: ```suggestion
The D-directorate is the board of the Computer Science chapter and consists of nine members.
```

for educational issues, student welfare and equality, student activities, and communication and business relations. Together they arrange Chapter Meetings, make sure the decisions at chapter meetings are executed, coordinate different parts of the chapter, and make strategic and long term plans.

Elections for board members are made in the same way as for officials, except for Chairman, Vice Chairman, and Treasurer.

[More information about the board members.](/sektionen/sammansattning?lang=en)

### Clubs

A club is an organisational entity created for a specific purpose. One of our clubs monitors our study environment, another takes care of the chapter hall, another arranges parties and so forth. Every club has a chairman that is elected at the SM. How to become member of a club varies from club to club. Some clubs have a limit to how many members they bring in and some have
interviews. If you want to know more about a certain club, you are welcome to contact the club's chairman! You can find each club's chairman's contact information under the respective club's tab in [clubs](/namnder?lang=en).

> If you want to create your own club by sending a motion to the board of directors. At the next Chapter Meeting, all members vote on whether or not your proposal should pass. You should make a budget, a description of your activities, and possibly formulate changes in our by-laws/rules.

Reviewer: This formulation seems quite strange.
What is the "board of directors"?.
```suggestion
If you want to create your own club you do this by sending a motion to the Chapter Board. At the next Chapter Meeting, all members vote on whether or not your proposal should pass. You should make a budget, a description of your activities, and formulate changes in our by-laws/rules to define the club.
```

It's also possible to make a project in the chapter. A project is similar to a temporary club, it is created the same way, with a motion.

### Officials

Officials are also elected at a chapter meeting. Every official is responsible for one club or activity in the chapter. If you want to get engaged in a club or activity you are welcome to turn to its volunteer!

> If you want to be a volunteer you have to run for election. Most of our elections are handled at the Election Chapter Meeting (mid May) and Mulled Wine Chapter Meeting (Late November/early December) and the nomination period starts rougly one month before the meetings. To nominate and handle your nominations go to our [election page](val.datasektionen.se). If you are interested in knowing what the responsible of a position is, you can contact
> the current volunteer.

Reviewer: Inconsistent terminology
```suggestion
If you want to be a official you have to run for election. Most of our elections are handled at the Election Chapter Meeting (mid May) and Mulled Wine Chapter Meeting (Late November/early December) and the nomination period starts roughly one month before the meetings. To nominate and handle your nominations go to our [election page](val.datasektionen.se). If you are interested in knowing what the responsible of a position is, you can contact
the current official.
```
